### PR TITLE
Moves sponsors to bottom

### PR DESCRIPTION
**Pull request by bridgetkromhout**
I think @mattstratton is right in https://github.com/devopsdays/devopsdays-web/issues/295 and we should approach the re-design in a considered fashion. When I started looking at what it would take to move the sponsors to the bottom, though, it was so trivial that I figured I'd throw a PR together for consideration.

I ran through all pages on all 2016 events to make sure this change wouldn't make them unusable. There are a few cosmetic issues caused by how non-responsive the site is in general, but https://github.com/devopsdays/devopsdays-web/commit/a1709011e286bf547ed7267a51d29ad3163b700b fixes the only major issue.

Because moving sponsors to the bottom means there is a lot more space for them, I went with what I thought was more aesthetically pleasing of no borders (https://github.com/devopsdays/devopsdays-web/commit/d6b82c510a980b62d0fa7e40a92f3a6f971fdc5d). Here's what that looks like in practice:

screen shot 2016-05-14 at 5 03 32 pm

There are improvements we could make in the future (perhaps centering all this) that should definitely wait for a redesign.

I also widened the 2016 programs as seen in these before & after screencaps. While it could be argued that should wait for the redesign, it looked really weird without the sponsor sidebar since it didn't move to fill the space.

Before:
screen shot 2016-05-14 at 5 12 49 pm

After:
screen shot 2016-05-14 at 5 13 12 pm

Let the bikeshedding commence. :) (Do not merge without @mattstratton's 👍 .)

### CODE REVIEW

**Comment by kmugrage** [comment]
This looks much better. 

(Sponsor hat on) When sponsoring, I generally try to get my AP to pay as quickly as possible so that our logo is "above the fold". In this layout, that's never going to happen, so the exposure is a bit lower. I'm personally ok with the negligible impact, but others may have larger concerns.

**Comment by MikeRosTX** [comment]
I personally like the esthetics of the pages because there's more real estate to the right to accommodate the schedule and other items.

What I'm afraid of are the complaints coming from sponsors stating that people might not scroll all the way down the page to see their logo and who is supporting/sponsoring the particular event at each city.

**Comment by yvovandoorn** [comment]
So screenshot for after doesn't show what happens to the actual sponsor. How does prioritizing sponsor levels work.

**Comment by mattstratton** [comment]
@yvovandoorn - see the earlier screenshot 

https://cloud.githubusercontent.com/assets/2104453/15270924/edcce328-19f6-11e6-988e-d421d03933d3.png

**Comment by bridgetkromhout** [comment]
To address @yvovandoorn's question:

A view of the "after" Minneapolis main page "above the fold" when zoomed all the way out:

screen shot 2016-05-14 at 8 29 08 pm

Scrolled down to show all the Minneapolis sponsors:

screen shot 2016-05-14 at 8 28 58 pm

The Amsterdam front page (zoomed all the way out):

screen shot 2016-05-14 at 8 31 04 pm

Scrolled down, the rest of the Amsterdam sponsors:

screen shot 2016-05-14 at 8 31 37 pm

For both Minneapolis and Amsterdam I'd probably recommend a div to rearrange the welcome page a bit, take better advantage of the extra space horizontally.

I understand sponsors wanting visibility - I work for a sponsor too - and I also know the the reality of most conference websites is that the sponsors are in a footer on every page (at best) or tucked away on a sponsor page (at worst). And as we saw in the complaints about how the sidebar would scroll off to the bottom, we're building a confusing expectation with the current layout (see @whatschrisdoing's comment in https://github.com/devopsdays/devopsdays-web/issues/295#issuecomment-219218290).

Also an argument could be made that being in a fixed-width sidebar on the right is dreadful for mobile, since you'll never be seen at all if people are zoomed in enough to read.

@kmugrage is right that we may have created an artificial economy of rushing for placement on the page - but recall, the old site also would shuffle the icon placement inside a level! which we're no longer doing.

**Comment by mattstratton** [comment]
Honestly the right solution is to make it responsive. At a larger breakpoint the sponsors will be on the right and a smaller breakpoint they go in the bottom.

**Comment by bridgetkromhout** [comment]
> Honestly the right solution is to make it responsive. At a larger breakpoint the sponsors will be on the right and a smaller breakpoint they go in the bottom.

Agree. And that's not something we'll solve today (as nice as that would be), since we do need to work with a designer.

The problems I'd like to solve right now are:
* difficult to read the narrow-column program
* sponsor icons look cramped
* difficult to see all the sponsors of a given level together because only two (or three if we change that) fit per row
* difficult to do bigger sponsor icons for higher levels (this doesn't solve it but it sets us up to solve it)

And the reason this is currently bugging me more than it did before:
* video embeds of a fixed size like those generated by vimeo spill over onto the sponsors (ignore the lack of permissions to embed - London knows and says they'll fix it)

screen shot 2016-05-14 at 9 20 15 pm

**Comment by MikeRosTX** [comment]
I am definitely OK with having sponsors at the bottom. As long as we all agree to inform sponsor that complain, that we're just aligning up with how other conference sites publish sponsors listed at the footer.

Agree with Bridget regarding video embedding so close to the sponsor logos would bug me too. Definitely having sponsors at the footer is better to view schedules, embedded videos, Eventbrite, etc.

**Comment by jedi4ever** [comment]
I agree with @kmugrage - logo above the fold is what the logo on the site is worth - if you're a smalller event it's one of the few extras you get. Moving it below the fold is less valuable.

Given screens are wider now I'd suggest keeping it right and only moving it below the fold if on a mobile device or smaller screen estate

**Comment by yvovandoorn** [comment]
:+1: @kmugrage and @jedi4ever.

Keeping it on the right in order of level is something a majority of our sponsors expect. In most situations we have the screen real estate these days to have it in the right.

**Comment by MikeRosTX** [comment]
I completely understand what @kmugrage, @jedi4ever & @yvovandoorn are stating about keeping it on the right side visible and valuable to the sponsors.

But if we clearly explain to those sponsors who complain that we're just trying to make DevOpsDays website look similar to the way other major conferences look that have the sponsors at the bottom, I'm sure they'll understand.

It's better to ask for forgiveness than their permission up front.  If push came to shove and we had a MAJORITY of the sponsors raise a lot of complaints, we can always change it back.  But we should at least give it a try and to those sponsors who complain, we can point the to these big conference to show them that all sponsors are at the bottom of the screen.

http://events.itrevolution.com/us/sponsors/

https://ignite.microsoft.com/#fbid=CidXhYwSAu1

http://conferences.oreilly.com/oscon/open-source-us

**Comment by mattstratton** [comment]
Also please see #359 where I will be working to make this responsive.

**Comment by yvovandoorn** [comment]
TL;DR - The sponsors (for a significant part) fund the event, the community & organizers makes the program happen. Shoving sponsors to the bottom where screen real estate allows for them to co-exist seems unnecessary devalues for a sponsor to sponsor (and thus could impact how we reach the community).

@MikeRosTX so this is my 1st year part of devopsdays, but one of the most refreshing things is that devopsdays *isn't* like other conferences. Personally, and this is my 2 cents, I don't want devopsdays to be compared to ignite, velocity, oscon, chefconf or others. 

The right side to me has always felt natural. The way we often position sponsors at these events is that their sponsorship keeps ticket costs down, thus accessibility high towards new attendees.

**Comment by yvovandoorn** [comment]
Thinking more about it -- this should be a layout choice that each event can make as they setup their site versus a global enforced change. 

The default staying how it is (at least for 2016) and if certain cities decide to go with the new layout, they can declare that in `data/layout/events/year-city.yml` file as `sponsors: bottom`

**Comment by mattstratton** [comment]
The challenge is that on a smaller screen you do no have the real estate for a sidebar. Which is why on smaller screen it should break to being at the bottom but stay in the side for larger break points. See the linked reposive issue.

**Comment by MikeRosTX** [comment]
@yvovandoorn, this is also my 1st year as local organizer of devopsdays.  But I like your idea of "_this should be a layout choice that each event can make as they setup their site versus a global enforced change._"

**Comment by yvovandoorn** [comment]
@mattstratton I completely get the mobile impact and completely agree that on a mobile device, content (program info) wins over trying to share it with sponsors.

But any device doing a native resolution of 1024x768 will properly render the page.

**Comment by mattstratton** [comment]
Yes that is my design idea.

**Comment by mattstratton** [comment]
Also, with regard to the video overflow, by enforcing responsive stuff that I have in mind, these things can be protected against.

**Comment by mattstratton** [comment]
Replying to @bridgetkromhout statement that we cannot implement responsive until we have a designer - not necessarily. I can implement the same look and feel (no change to UX) but make it responsive without a designer. We need a designer to refactor the overall look and feel. I think I can convert what we have now to a responsive layout which will solve for these issues but not change the user experiece overall.

**Comment by WhatsChrisDoing** [comment]
@mattstratton Would the responsive layout allow the sponsors to fit on the right more often on the home page? Would it make more sense to just have the sponsors on the home page and not on all of the other pages?

**Comment by mattstratton** [comment]
I don't understand what "more often on the right" means? Can you clarify?

**Comment by WhatsChrisDoing** [comment]
That the sponsor logos would display on the right side of the page at narrower page widths. Currently the sponsor logos wrap to the bottom when the window is about 990 pixels on my Mac. It's the wrong question. I think our event logo would drive the minimum column size in any case.

**Comment by barkerd427** [comment]
I'd vote for reactive. We'll always have new events, and part of their value to sponsors is page presence. I'd also be ok with sponsors only being on the event homepage.

**Comment by mattstratton** [comment]
I think that having the sponsors display on the right at a narrow viewport makes for a really bad user experience. It makes everything squished.

**Comment by WhatsChrisDoing** [comment]
Likely true. I think where I'm landing after this discussion is to stay with the status quo for now and execute a home page redesign that better accommodates the sponsor logos at typical tablet portrait and landscape screen width.

**Comment by iennae** [comment]
I think having sponsors appear on every page on the bottom is great. On the sponsors page making them front and center is awesome too. Not every conference puts sponsors on every single page. I think the schedule (as well as embedded videos) much nicer if things weren't crammed all to the left. Long titles, and times where multiple things are happening get smushed so it's not as nice. 

Based on the fold that Bridget shows, it looks like platinum (or whichever top sponsor) will get premium space as well which also will help folks in selecting for that visibility.

**Comment by iennae** [comment]
Also.. is this a bug??
When the page is bigger the sponsors move up a level and squish the menu. 

Smaller Window - Doesn't Squish menu
Larger Window - Squishes local menu

Originally I wanted to point out from a user point of view, this is a lot of content before we actually get to the content around the schedule.

**Comment by bridgetkromhout** [comment]
I'm closing this unmerged for now. We can keep discussing what it should look like in https://github.com/devopsdays/devopsdays-web/issues/295. Thanks, all.
